src/test/shell/data/i18n/en.json

```diff
@@ -254,6 +254,11 @@
     "HIDE_UNDEPLOYED": "Hide undeployed instances",
     "UNDEPLOYED_VISIBLE": "Show undeployed instances",
     "SORT": "Sort",
+    "MANUAL_UPDATE" : {
+      "HEADER" : "Manual version upgrade",
+      "BODY" : "Manual update instance version information. Select correct version below",
```

Comment:
Update the current application version for this application instance based on an upgrade process that was performed manually. Select the new correct version from the list below.